tests/Remove-DbaDbMailAccount.Tests.ps1

```diff
@@ -0,0 +1,57 @@
+$CommandName = $MyInvocation.MyCommand.Name.Replace(".Tests.ps1", "")
+Write-Host -Object "Running $PSCommandPath" -ForegroundColor Cyan
+. "$PSScriptRoot\constants.ps1"
+
+Describe "$CommandName Unit Tests" -Tag 'UnitTests' {
+    Context "Validate parameters" {
+        [array]$params = ([Management.Automation.CommandMetaData]$ExecutionContext.SessionState.InvokeCommand.GetCommand($CommandName, 'Function')).Parameters.Keys
+        [object[]]$knownParameters = 'SqlInstance', 'SqlCredential', 'Name', 'ExcludeAccount', 'InputObject', 'EnableException'
+        It "Should only contain our specific parameters" {
+            Compare-Object -ReferenceObject $knownParameters -DifferenceObject $params | Should -BeNullOrEmpty
+        }
+    }
+}
+
+Describe "$CommandName Integration Tests" -Tag "IntegrationTests" {
+
+    BeforeEach {
+
+        $server = Connect-DbaInstance -SqlInstance $script:instance2
+        $server2 = Connect-DbaInstance -SqlInstance $script:instance3
```

Comment:
We can just test this against instance3 to save on processing time, no real difference testing between 2016 and 2017 anyway.

Reply:
The purpose of this test is to check if the command works against multiple instances, as there were commands in the past where supplying multiple instances did not work, i.e. command worked only on the last instance from the list.

Reply:
Removed the second server to save the processing time.